code/game/gamemodes/clock_cult/clock_cult.dm

```diff
@@ -130,6 +130,7 @@ Credit where due:
 	required_enemies = 4
 	recommended_enemies = 4
 	enemy_minimum_age = 14
+	round_ends_with_antag_death = 1
```

Comment:
TRUE

Reply:
![image](https://user-images.githubusercontent.com/16315400/36360974-7ba46b48-14f6-11e8-96cf-9997f6cb8120.png)

Reply:
Wizard is old code. `TRUE` is the same thing as `1` and is the modern standard for true/false variables like this one is.